interface/main/left_nav.php

```diff
@@ -1118,6 +1208,23 @@ function selpopup(selobj) {
 
     </ul>
   </li>
+  <?php if($GLOBALS['enable_group_therapy']) : ?>
+      <li class="open"><a class="expanded" id="patimg" ><i class="fa fa-fw fa-users"></i>&nbsp;<span><?php xl('Group','e') ?></span></a>
```

Comment:
lets not expand this by default or it causes weird initial behavior. If click to close this or the patient item, then they both close.